.github/workflows/lint.yml

```diff
@@ -0,0 +1,27 @@
+name: Lint
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7.0
+        bundler-cache: true
+    - name: Run linters with autofix
+      run: bundle exec rubocop --auto-correct-all
+    - name: Push any fixes
+      run: |
+        git config --global user.email "lint-bot@example.com"
```

Comment:
We should probably not leave an `@example.com` address in there, but it doesn't have to be a real email address either. How about just `git config --global user-email "lint-bot"`? I tried an experiment and git doesn't seem to check it's a valid email address.

Reply:
Good idea!